README.md

```diff
@@ -33,7 +33,7 @@ Users can:
 # Development
 
 ## Rust
-* Built and developed using - rust stable(`rustc 1.57.0 (f1edd0429 2021-11-29)`)
+* Built and developed using rust toolchain `nightly-2022-04-01`
```

Comment:
I'd argue against requiring a nightly toolchain. To make selection of the right compiler automatic, we added a rust-toolchain.toml file in mango-v4: https://github.com/blockworks-foundation/mango-v4/blob/dev/rust-toolchain.toml

Maybe use 1.65 too, that version hasn't yet caused us trouble over there.

Reply:
1.65 looks good, updated with same toolchain file